src/main/java/uk/gov/laa/ccms/caab/controller/submission/ClientCreateSubmissionConfirmedController.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.laa.ccms.caab.controller.submission;
+
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.APPLICATION_DETAILS;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_INFORMATION;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_SEARCH_CRITERIA;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.USER_DETAILS;
+import static uk.gov.laa.ccms.caab.constants.SubmissionConstants.SUBMISSION_CREATE_CLIENT;
+
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.SessionAttribute;
+import uk.gov.laa.ccms.caab.bean.ApplicationDetails;
+import uk.gov.laa.ccms.caab.service.ApplicationService;
+import uk.gov.laa.ccms.data.model.UserDetail;
+import uk.gov.laa.ccms.soa.gateway.model.ClientDetail;
+
+/**
+ * Controller for confirmed create client submissions.
+ */
+@Controller
+@RequiredArgsConstructor
+@Slf4j
+public class ClientCreateSubmissionConfirmedController {
+
+  private final ApplicationService applicationService;
+
+  /**
+   * Handles the POST request for a client creation submission.
+   *
+   * @param clientInformation The client details from the session
```

Comment:
Move clientInformation param to correct position

Reply:
moved